.github/workflows/pants.yaml

```diff
@@ -30,10 +30,24 @@ jobs:
       id: cache
       with:
         path: |
-          ~/.cache/pants/setup
-          ~/.cache/pants/lmdb_store
-          ~/.cache/pants/named_caches
-        key: ${{ runner.os }}-
+          ~/.cache/pants/setup # The initial bootstrapping of Pants
+          ~/.cache/pants/named_caches  # Caches used by underlying tools like Pip and PEX.
+          # If you're not using a fine-grained remote caching service, such as the one provided by
+          # Toolchain(https://toolchain.com/), the lead sponsor of Pants, then you may also want to
+          # preserve the local Pants cache:
```

Comment:
Can we include the alternative, off-the-shelf, options as well?
https://github.com/bazelbuild/remote-apis lists the tools and companies (you should PR Toolchain to be on that list :stuck_out_tongue: )

I'd be worried people don't know they can also try hosting their own if partnership is prohibitive or scary, or want to PoC themselves. Still rep Toolchain here but let people know where to look for extra.

Reply:
Wait, we even have a doc for this, lol; `docs/markdown/Using Pants/remote-caching-execution.md`

Reply:
Yeah I will link to that page.